samples/outbox/store-and-forward/Version_5/Sender/app.config

```diff
@@ -1,17 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
it depends what the main focus of the sample is. generally that can be answered by asking this. if, within the same major of the core, there are multiple majors of a downstream, what would happen in this sample. the answer is either "we would still only target a single major of the core" or "we would want a new version for the major of the downstream"